source/module_esolver/esolver_ks_lcao_tddft.cpp

```diff
@@ -48,17 +48,17 @@ ESolver_KS_LCAO_TDDFT::~ESolver_KS_LCAO_TDDFT()
     {
         for (int ik = 0; ik < kv.nks; ++ik)
         {
-            delete Hk_laststep[ik];
+            delete[] Hk_laststep[ik];
```

Comment:
Excellent! I am curious how did you catch this bug?

Reply:
> Excellent! I am curious how did you catch this bug?

These bugs are shown in the [dynamic analysis report](https://deepmodeling.github.io/abacus-develop/).